src/lib.rs

```diff
@@ -562,6 +696,44 @@ impl PublicKey {
         Ok(sig)
     }
 
+    pub fn finalize_with_metadata(
```

Comment:
Docstring here, too, and for all of the new API methods.

Reply:
done